blocksuite/framework/store/src/model/workspace.ts

```diff
@@ -15,6 +16,8 @@ export interface Workspace {
   readonly idGenerator: IdGenerator;
   readonly blobSync: BlobEngine;
   readonly awarenessStore: AwarenessStore;
+  readonly onLoadDoc?: (doc: Y.Doc) => void;
+  readonly onLoadAwareness?: (awareness: Awareness) => void;
```

Comment:
we should use `Slot` to manage events in blocksuite.

Reply:
This api is more like a method than an event. I will refactor it later.